packages/manager/apps/dedicated/client/app/ip/byoip/byoip.controller.js

```diff
@@ -61,15 +61,15 @@ export default class IpByoipConfiguration {
       .then((token) => {
         this.token = token;
       })
-      .catch((err) =>
-        this.Alerter.error(
+      .catch((err) => {
+        return this.Alerter.error(
```

Comment:
You can use the expression body instead of the block body as there is nothing more than a return on the block